src/routes.tsx

```diff
@@ -94,6 +94,15 @@ const Routes = ({ token }) => (
       ))
     }
 
+    <Route
+      path="/info"
+      component={() => {
+        // Redirect removed route
+        window.location.href = 'https://www.fixmycity.de/wer-wir-sind';
+        return null;
+      }}
+    />
+
```

Comment:
Ich denke, das ist besser in https://github.com/FixMyBerlin/fixmy.frontend/blob/develop/_redirects aufgehoben. Dort ist es dann ein 301 redirect, was für SEO die bessere Lösung ist. Ich würde es auch hier in den Routes nicht erwarten.

Reply:
Ja, da hast du recht! Das kannte ich noch nicht :)